sdk/core/Azure.Core/src/DynamicData/MutableJsonChange.cs

```diff
@@ -139,6 +142,7 @@ internal bool IsGreaterThan(ReadOnlySpan<char> otherPath)
             return Path.AsSpan().SequenceCompareTo(otherPath) > 0;
         }
 
+        [RequiresUnreferencedCode(SerializationRequiresUnreferencedCode)]
```

Comment:
Curious: what reflection is used in this method?

I know I refactored type this while you were working on this PR, so I'm wondering if this is left over from before?

If it's not (happy to learn 🙂), it's a goal we have for this type to be compatible with AOT/trimming, so I'd like to understand what we'll need to do to get there, and make sure we can remove this attribute in the future when we do.

Reply:
This attribute is not needed anymore after the refactoring! Thank you for catching it ☺️